assets/src/dashboard/components/cardGridItem/cardPreview.js

```diff
@@ -64,28 +64,15 @@ const EditControls = styled.div`
   justify-content: space-between;
   padding: 0;
   transition: opacity ease-in-out 300ms;
-  background: ${({ theme }) => theme.DEPRECATED_THEME.cardItem.previewOverlay};
-  border-radius: ${({ theme }) =>
-    theme.DEPRECATED_THEME.storyPreview.borderRadius}px;
+  background: ${({ theme }) => theme.colors.opacity.white16};
+  border-radius: ${({ theme }) => theme.borders.radius.small};
   opacity: ${({ isActive }) => (isActive ? 1 : 0)};
 
-  ${KEYBOARD_USER_SELECTOR} &:focus {
-    outline: ${({ theme }) =>
-      `2px solid ${rgba(
-        theme.DEPRECATED_THEME.colors.bluePrimary,
-        0.85
-      )} !important`};
-  }
-
-  @media ${({ theme }) => theme.DEPRECATED_THEME.breakpoint.smallDisplayPhone} {
-    button,
-    a {
-      min-width: ${({ cardSize }) => cardSize.width};
-      & > label {
-        font-size: 12px;
-      }
+  ${({ theme }) => css`
+    &[${ThemeGlobals.FOCUS_VISIBLE_DATA_ATTRIBUTE}] {
```

Comment:
This is interesting! So we're forcing focus on grid items by an invisible button to control a11y while maintaining the css grid + absolutely rendered previews, anywho the focus visible class was sometimes not getting applied but the data attribute always is.

Reply:
that's weird. Should we add the `FOCUS_VISIBLE_DATA_ATTRIBUTE` selector to our global focus css so it covers this case as well?

Reply:
It feels like it shouldn't be necessary? The way the grid hijacks focus is unique/one off. Let's keep eyes on it and see?